src/v/utils/base64.h

```diff
@@ -46,3 +46,5 @@ ss::sstring iobuf_to_base64(const iobuf&);
 /// \throws base64_url_decoder_exception if an invalid URL base64 encoded string
 /// is provided
 bytes base64url_to_bytes(std::string_view);
+
+iobuf base64_decode(const iobuf&);
```

Comment:
nit: the other base64 encode/decode functions are styled as a `buffer_to_buffer`.  Maybe this should be `base64_to_iobuf` or something?

Reply:
Changed